Dockerfile

```diff
@@ -1,6 +1,36 @@
+ARG BUILD_TYPE
 ARG CUDA_VERSION
+ARG PYTHON_VERSION=3.10
```

Comment:
`ARG PYTHON_VERSION=3.10` currently appears in 3 places in the file. Some of those might be redundant.

Reply:
Yes, the first one can be removed.  Those are (sub-)image specific variables so the last two need to be there.  I'll push that change later (to avoid triggering the CI right now).